MaxMind.Db/Reader.cs

```diff
@@ -247,7 +252,10 @@ private int StartNode(int bitLength)
         private T Deserialize<T>(JToken value)
         {
             var serializer = new JsonSerializer();
-            return serializer.Deserialize<T>(new JTokenReader(value));
+            using (var reader = new JTokenReader(value))
+            {
+                return serializer.Deserialize<T>(reader);
+            }
         }
```

Comment:
Cool. Did a source analyzer discover this? I missed it through a couple of reviews I guess.